wger/core/views/user.py

```diff
@@ -40,6 +43,11 @@
 from django.conf import settings
 from rest_framework.authtoken.models import Token
 
+from fitbit import FitbitOauth2Client
+import base64
+import requests
```

Comment:
You can move the import statements here to the top.

Reply:
Why should he do it yet flake 8 has not raised any complaints.

Reply:
This is just to conform to pep8 standards. According to [this](https://pep8.org/#imports) resource, import statements should be grouped as follows:

1. standard library imports
2. related third party imports
3. local application/library specific imports